pkg/config/process.go

```diff
@@ -73,9 +76,8 @@ func setupProcesses(config Config) {
 	procBindEnvAndSetDefault(config, "process_config.container_collection.enabled", true)
 	procBindEnvAndSetDefault(config, "process_config.process_collection.enabled", false)
 
-	config.BindEnv("process_config.process_dd_url", "")
+	procBindEnvAndSetDefault(config, "process_config.process_dd_url", DefaultProcessEndpoint)
```

Comment:
there's a `DD_PROCESS_AGENT_URL` override in `loadEnvVariables()`. Should we move it to here?

Reply:
Oh yeah, thanks for pointing that out!